### DISCUSSION TITLE
dependency-track site flagged by google as Phishing

### DISCUSSION THREAD

**Discussion by danielpof** [DependencyTrack/dependency-track#1450] [Q&A] [answered]
Just setup dependency-track for my organization and on a public URL for devs to access and it keeps getting flagged by google and Microsoft as a Phishing site.

Anyone else being faced with this issue?

**Comment by stevespringett** [comment, marked as answer]
When you say its being flagged by google and Microsoft, do you mean by the Chrome and Edge browsers? The search engines (google.com and bing.com) should not even display it.

If you're referring to the browsers, then its likely a certificate issue. You'll need to ensure you have TLS properly configured with a certificate from a trusted CA.

**Reply by danielpof** [reply]
I mean the browser based protection screen (MS Smart Defender and google's equivalent), ironically only the frontend gets flagged, the API is fine even though both are setup the same way on the same reverse proxy.

I did do an ssllabs cert validation and get an A rating with an auto-provisioned let's encrypt certificate with nothing that caught my eye as wrong and a final A rating.

What's frustrating is that I have no idea why it's being flagged and reporting the site as safe has no effect making it unusable as the browser Alert breaks OpenID login.

**Reply by stevespringett** [reply]
I have no idea. I haven't seen or used Windows in years and I don't think I know what Google's equivalent is. But I've never seen it. It's possible that both Microsoft and Google use the same third-party data/service to help determine this. You might want to raise an issue with Microsoft and Google to get clarity. 

According to https://feedback.smartscreen.microsoft.com/smartscreenfaq.aspx

> Our goal is to minimize false warnings or blocks. In the rare case of a false warning, we offer a web-based feedback system to help users and website owners report any errors as quickly as possible. These reports are verified by our support team and mistakes are corrected.

But they do not provide the url to the feedback system. Perhaps it's part of Smart Defender?

**Reply by danielpof** [reply]
Yes, google provides the data for firefox at least (it also shows the warning) and I assume Microsoft Smart Screen simply includes it as well.

You can see the error on this Microsoft demo site: https://demo.smartscreen.msft.net/phishingdemo.html

According to [them](https://demo.wd.microsoft.com/Page/UrlRep) it's for _"a page that known for phishing that should be blocked"_, which seems odd for a site that exists for less then a week.

Anyways if nobody else is having the issue then I can only assume something else is happening that's unrelated with dependency-track. 

Thanks for the help
